.github/workflows/sourcegraph.yml

```
@@ -5,19 +5,19 @@ on:
       - main
   pull_request:
 jobs:
-  lsif:
+  sourcegraph:
     runs-on: ubuntu-latest
-    name: "Upload LSIF"
+    name: "Upload Sourcegraph Code Intel"
     steps:
       - uses: actions/checkout@v2
       - uses: coursier/setup-action@v1.1.2
         with:
           jvm: adopt:8
-          apps: lsif-java
-      - run: lsif-java index
-      - name: Upload LSIF data
-        uses: sourcegraph/lsif-upload-action@master
-        with:
-          endpoint: https://sourcegraph.com
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          file: dump.lsif
+      - name: cs launch scip-java
+        run: |
+          cs launch com.sourcegraph:lsif-java_2.13:latest.stable -M com.sourcegraph.lsif_java.LsifJava -- index
```

```suggestion
          cs launch com.sourcegraph:scip-java_2.13:latest.stable -M com.sourcegraph.scip_java.ScipJava -- index
```
I confirmed it works in my local environment